Source/42fsw.c

```diff
@@ -1687,12 +1687,7 @@ void FlightSoftWare(struct SCType *S)
       struct IpcType *I;
       long Iipc;
       #endif
-      
-      if (S->InitAC) {
-         S->InitAC = 0;
-         InitAC(S);
```

Comment:
This same initialization is already being done at the appropriate location, so this extra call just needs to be removed. Other `InitAC()` call at `42init.c`